libfwupdplugin/fu-usb-endpoint.c

```diff
@@ -0,0 +1,322 @@
+/*
+ * Copyright 2020 Emmanuel Pacaud <emmanuel@gnome.org>
```

Comment:
Presumably you'll want to update this

Reply:
I wasn't sure what to do here; the original code was written in 2020 by Emmanuel and all I really did was change the GUsb prefix to FuUsb and ported it to use the FwupdCodec stuff -- it seemed wrong to say that Emmanuel wrote it in 2024 and it doesn't seem right to say I wrote it either. I've added my copyright (for this year) _additionally_ -- is that better?

Reply:
I'd say the right answer is to leave the existing one in 2020 for original author and add a new line for you for 2024.